libs/dh/market-participant/actors/feature-delegation/src/lib/util/dh-group-delegations.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { DelegationMessageType } from '@energinet-datahub/dh/shared/domain/graphql';
+
+import {
+  DhDelegations,
+  DhDelegationsByDirection,
+  DhDelegationsByType,
+  DhDelegationsGrouped,
+} from '../dh-delegations';
+
+const outgoingTypes = [DelegationMessageType.Rsm016Outbound, DelegationMessageType.Rsm017Outbound];
+const incomingTypes = [
+  DelegationMessageType.Rsm012Inbound,
+  DelegationMessageType.Rsm014Inbound,
+  DelegationMessageType.Rsm016Inbound,
+  DelegationMessageType.Rsm017Inbound,
+  DelegationMessageType.Rsm019Inbound,
+];
+
+export function dhGroupDelegations(delegations: DhDelegations): DhDelegationsGrouped {
```

Comment:
Question: Why the dh prefix?

Reply:
Maybe an just the habit of adding "dh-" prefix to file names, classes, types, etc to things that are public for others to use :)